helm/wrongsecrets-ctf-party/templates/wrongsecrets-balancer/deployment.yaml

```diff
@@ -19,13 +19,11 @@ spec:
         app: wrongsecrets-balancer
         {{- include "wrongsecrets-ctf-party.selectorLabels" . | nindent 8 }}
     spec:
-      securityContext:
-        runAsUser: 1000
-        runAsGroup: 3000
-        fsGroup: 2000
-        seccompProfile:
-          type: RuntimeDefault
```

Comment:
Does this still effectively work ? 
In contrast to the namespaced things (WrongSecrets & WrongSecrets desktop), I rather have this defined at deployment template level and with its default values in the helm chart values.

Reply:
Yes, it is.
If you look at the generated `test-tmp.yaml`  you can notice that these values are present. 
https://github.com/OWASP/wrongsecrets-ctf-party/pull/249/files#diff-0e79142774e4871080c08cade4299253e8b84c26fab30421555d6a454406abf7R241

All I did is to make the values configurable by the `values.yaml` file under the section `podSecurityContext`